.storybook/main.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  stories: ['../assets/js/src/**/_stories/*.js'],
```

Comment:
Shouldn't we rather use typescript for stories? It is a new feature so I guess we could even force TS for this. We will probably write components in TS as well.
This will probably require some additional configuration https://storybook.js.org/docs/configurations/typescript-config/ but think we need it.